codalab/worker/bundle_manager.py

```diff
@@ -329,6 +329,12 @@ def _try_start_bundle(self, workers, worker, bundle):
         """
         if self._model.set_starting_bundle(bundle, worker['user_id'], worker['worker_id']):
             workers.set_starting(bundle.uuid, worker)
+            if self._worker_model.shared_file_system and worker['user_id'] == self._model.root_user_id:
```

Comment:
Why does the user have to be root here?
This doesn't parallel the code in run.py.